generator/uscore/uscore_generator.rb

```diff
@@ -938,12 +938,10 @@ def create_multiple_or_test(sequence)
 
               reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
               validate_search_reply(versioned_resource_class('#{sequence[:resource]}'), reply, search_params)
-              assert_response_ok(reply)
```

Comment:
FYI: this is a redundant check, as we check this in `validate_search_reply` already.